index.html

```diff
@@ -150,7 +150,31 @@ <h6>languaages</h6>
                 </li>
                  </ul>
             </section>
+            <section class="contact">
+              <h1 class="me">Contact me</h1>
+              <p>If you have an application you
+                are interested in developing, a
+                feature that you need built or aproject that needs coding. 
+                
+                I’d love to help with it!</p>
+                <form class="contact_form">
+                     <label for=""></label>
+                    <input type="text" placeholder="Lorem Ipsum" value="" class="form1">
```

Comment:
- Nice work so far 👍🏿 Upon reviewing the provided code, I have identified a few areas that require attention:
- [ ] Missing maxlength attribute: The code does not include the "maxlength" attribute, which may be necessary to enforce a specific character limit on the input field. It is important to clarify if there is a maximum length requirement and add the appropriate "maxlength" value.
- [ ]  I can see that the "required" attribute is indeed not present in the provided code. The inclusion of the "required" attribute ensures that the user must fill out the input field before submitting the form, meeting the requirement for a required field.
- [ ] Incomplete type attribute: The "type" attribute is present, but its value is missing. The "type" attribute should be filled with the correct value to indicate the expected data format for the input field. It is crucial for proper data validation.
- [ ] [OPTIONAL] The placeholder value: The current placeholder value, "Lorem Ipsum," is generic and does not provide meaningful guidance to the user. It would be beneficial to replace it with a descriptive placeholder that indicates the expected input like `Enter full name`


Overall, the code structure and formatting are satisfactory. However, it is important to address these areas to ensure better user experience and adherence to the requirements.